k8s/gitlab/README.md

```diff
@@ -353,6 +353,117 @@ echo "http://${SERVICE_IP}/"
 
 Use your GitLab root password generated earlier to sign-in.
 
+# Backup
+
+Below steps are based on [Official GitLab Backup/Restore documentation](https://docs.gitlab.com/ee/raketasks/backup_restore.html#back-up-gitlab) .
+
+```
+APP_INSTANCE_NAME=gitlab-1
+NAMESPACE=default
+```
+
+Run Backup command:
+```
+kubectl --namespace ${NAMESPACE} exec -it \
+  "${APP_INSTANCE_NAME}-gitlab-0" -- bash -c "gitlab-backup create"
+```
+
+You can see created backup files from output of backup command or via running:
+```
+kubectl --namespace ${NAMESPACE} exec -it \
+  "${APP_INSTANCE_NAME}-gitlab-0" -- bash -c \
+  "ls /var/opt/gitlab/backups/"
+```
+
+Copy backup file to local workstation:
+```
+BACKUP_TAR="place-your-backup-file-here"
+
+kubectl --namespace ${NAMESPACE} cp \
+  ${APP_INSTANCE_NAME}-gitlab-0:/var/opt/gitlab/backups/${BACKUP_TAR} .
```

Comment:
Command did not work with .
To be consistent with later restore should it be like below? 

```
...
${APP_INSTANCE_NAME}-gitlab-0:/var/opt/gitlab/backups/${BACKUP_TAR} ${BACKUP_TAR}
```